e2e-report/components/table.js

```diff
@@ -1,30 +1,117 @@
-export default function Table({ th, name, suitesTotal, total, passed, retries }) {
+import Link from 'next/link'
+import { GithubIcon } from './icons'
+
+// Show test suites and non-passing cases per each
+export default function Table({ suites }) {
+  const countColumns = ['Pass', 'Fail', 'Known', 'Skip']
+
   return (
-    <table>
-      <tbody>
-        <tr>
-          {th.map((header) => (
-            <th key={header}>{header}</th>
-          ))}
-        </tr>
+    <table className="table issues-table text-base w-full">
+      <thead>
         <tr>
-          <td>
-            <h4>{name}</h4>
-          </td>
-          {suitesTotal && (
-            <td>
-              <h4>{suitesTotal}</h4>
-            </td>
-          )}
-          <td>
-            <h4>{total.toLocaleString()}</h4>
-          </td>
-          <td>
-            <h4>{Math.round((passed / total) * 100)}%</h4>
-            {retries > 0 ? <span className="retries"> (⚠️ retries: {retries})</span> : null}
```

Comment:
I imagine you intentionally removed this because it isn't great information to show publicly? It's somewhat useful for us. I wonder how we could hide it from the public while making it accessible to us?